session/handler.go

```diff
@@ -194,6 +198,89 @@ func (h *Handler) logout(w http.ResponseWriter, r *http.Request, ps httprouter.P
 	w.WriteHeader(http.StatusAccepted)
 }
 
+// swagger:parameters adminIdentitySession
+// nolint:deadcode,unused
+type adminIdentitySession struct {
+	// ID is the identity's ID.
+	//
+	// required: true
+	// in: path
+	ID string `json:"id"`
+}
+
+// swagger:model successfulAdminIdentitySession
+// nolint:deadcode,unused
+type AdminIdentitySessionResponse struct {
```

Comment:
Okay, so this stuff is an autogenerated client code for the resource you've added, correct?

Reply:
This is created by me for documentation purposes